app/controllers/admin/export/document_controller.rb

```diff
@@ -27,6 +27,13 @@ def unlock
     document.update!(locked: false)
   end
 
+  def migrated
+    document = Document.find(params[:id])
+    unless document.locked?
+      respond_with Hash.new, status: 422
```

Comment:
It seems a bit unusual to use `Hash.new` rather than `{}`. Though I'd wonder if we can just do a head method rather than needing a respond with and have it return empty content.

I'd suggest doing `head :unprocessable_entity` instead. 

From [a little reading](https://stackoverflow.com/questions/16133923/400-vs-422-response-to-post-of-data) I'm also wondering if :bad_request might be more suited. There isn't any issues with the syntax concerns here which lead it slightly more towards 422 but 400 does make it quite distinct from Rails validation where my concern lies there.